containers/themes/ThemesSection.js

```diff
@@ -44,11 +45,26 @@ const ThemesSection = () => {
         createModal(<CustomThemeModal theme={customCSS} onSave={handleSaveCustomTheme} />);
     };
 
-    const handleChangeTheme = async (themeIdentifier) => {
+    const handleChangeTheme = async (themeID) => {
         if (themeIdentifier === CUSTOM_THEME.identifier) {
+            await new Promise((resolve, reject) => {
+                createModal(
+                    <ConfirmModal
+                        submit={c('Action').t`Apply`}
+                        title={c('Title').t`Change theme`}
+                        onConfirm={resolve}
+                        onClose={reject}
+                    >
+                        <Alert type="warning">{c('Warning')
+                            .t`This action will permanently override your current theme. Are you sure you want to apply this new theme?`}</Alert>
```

Comment:
Maybe it would be better `This action will erase your current theme` ? So that it's even more clear that your theme would be lost?